### PR TITLE
Add Legendre 2001 ctation

### PR DESCRIPTION
**Pull request by artur-sannikov**
I added this [citation](https://bioconductor.org/packages/release/bioc/html/BiocBook.html).

For some reason, I have four commit but in reality only one file is changed.

### CODE REVIEW

File: inst/assets/bibliography.bib [hunk 1]
```diff
@@ -2139,3 +2139,14 @@ @article{Geistlinger2023
...
+
+@article{Legendre2001,
+  author = {Legendre, Pierre and Gallagher, Eugene D.},
+  title = {Ecologically Meaningful Transformations for Ordination of Species Data},
+  journal = {Oecologia},
+  month = oct,
+  volume = {129},
+  number = {2},
+  pages = {271--280},
+  url = {https://doi.org/10.1007/s004420100716},
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: could you instead change this to:

doi = {10.1007/s004420100716},

File: inst/pages/14_alpha_diversity.qmd [hunk 2]
```diff
@@ -108,16 +108,84 @@ plotColData(tse,
             "SampleType", 
             colour_by = "Final_Barcode") +
     theme(axis.text.x = element_text(angle=45,hjust=1)) + 
-  labs(y=expression(Richness[Observed]))
```
Comment (antagomir) [inline inst/pages/14_alpha_diversity.qmd:111]: The "labs" notation is nowadays the recommended way to do this. The ylab might be deprecated later. Could you switch back to "labs"?

**Review by antagomir — APPROVED** [review]
Approved with the comments.

File: inst/pages/14_alpha_diversity.qmd [hunk 3]
```diff
@@ -108,16 +108,84 @@ plotColData(tse,
...
+plots[[1]] + plots[[2]] +
+  plot_layout(guides = "collect")
+```
+ 
+Alternatively, the phylogenetic diversity can be calculated by `mia::estimateDiversity`. This is a faster re-implementation of   
+the widely used function in _`picante`_ [@R_picante, @Kembel2010].  
 
-Let us now compare the estimated Shannon and Faith indices across
-samples. First we add Shannon and Faith indices.
+Load `picante` R package and get the `phylo` stored in `rowTree`.
```
Comment (TuomasBorman) [inline inst/pages/14_alpha_diversity.qmd:188]: What is the intention of this? It seems that picante package is not loaded even though it says like that

Comment (artur-sannikov) [reply inst/pages/14_alpha_diversity.qmd:188]: I don't know. It's not something I have changed. That's a previous commit.

Comment (antagomir) [reply inst/pages/14_alpha_diversity.qmd:188]: For some reason it shows as part of this commit.

Have you synced your branch with the latest version? 

Anyway I thing that line could be removed.

Comment (artur-sannikov) [reply inst/pages/14_alpha_diversity.qmd:188]: I now figure out how to sync branches. It says "up to date".

Comment (TuomasBorman) [reply inst/pages/14_alpha_diversity.qmd:188]: Do you use normal git merge when updating with devel branch?

Comment (artur-sannikov) [reply inst/pages/14_alpha_diversity.qmd:188]: Used to. Now I usually use `git rebase`.

Comment (artur-sannikov) [reply inst/pages/14_alpha_diversity.qmd:188]: I will now just create new branches from devel and merge those directly on this repo. I think it's be easier.

Comment (TuomasBorman) [reply inst/pages/14_alpha_diversity.qmd:188]: rebase might explain this behavior

**Comment by artur-sannikov** [comment]
Closed in favor of #485.

**Pull request closed without merging**